custom_components/smartthinq_sensors/sensor.py

```diff
@@ -328,6 +339,40 @@ class ThinQSensorEntityDescription(SensorEntityDescription):
         unit_fn=lambda x: x.oven_temp_unit,
     ),
 )
+AIRPURIFIER_SENSORS: Tuple[ThinQSensorEntityDescription, ...] = (
+    ThinQSensorEntityDescription(
+        key=ATTR_PM1,
+        name="PM1",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_PM1,
+        value_fn=lambda x: x.pm1,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    ThinQSensorEntityDescription(
+        key=ATTR_PM25,
+        name="PM2.5",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_PM25,
+        value_fn=lambda x: x.pm25,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    ThinQSensorEntityDescription(
+        key=ATTR_PM10,
+        name="PM10",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_PM10,
+        value_fn=lambda x: x.pm10,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    ThinQSensorEntityDescription(
+        key=ATTR_FILTER_REMAINING_LIFE,
+        name="Fliter Remaining Life",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_BATTERY,
+        value_fn=lambda x: x.fliterRemainingLife,
+        native_unit_of_measurement=PERCENTAGE,
```

Comment:
May be is better to remove `DEVICE_CLASS_BATTERY` and specify a different icon (e.g. "mdi:air-filter")?